### PR TITLE
Add Quantum Scalar I6000 Tape Library

### PR DESCRIPTION
**Pull request by mpikzink**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by mpikzink** [comment]
Can someone help me to figure out why the tests fails? Do I have to update something?

**Comment by electrocret** [comment]
The step in the test units that are failing are the dev:check CI. From my understanding - this check puts the SNMPRec data provided into SNMPSim to simulate the device. It then has Libre discover the simulated device then compares what Libre found with control data that is captured in the JSON file. The fail output you're getting says there is a mismatch between what Libre is seeing from the simulation and the control data in the JSON.

@murrant or @Jellyfrog, can you confirm my understanding?

**Comment by murrant** [comment]
Correct @electrocret (maybe the docs need extra clarification).

Let's look at a little snippet:
```
-            'sensor_current' => 21
-            'sensor_limit' => null
-            'sensor_limit_warn' => 60
-            'sensor_limit_low' => null
-            'sensor_limit_low_warn' => 10
+            'sensor_current' => 21.0
+            'sensor_limit' => 41.0
+            'sensor_limit_warn' => 60.0
+            'sensor_limit_low' => 11.0
+            'sensor_limit_low_warn' => 10.0
```

Unfortunately, there is noise here.  For the code and tests 21 == 21.0, but it still shows up in the error output making it harder to read.

So the only changes here are `sensor_limit` and `sensor_limit`, they are now 41 and 11 respectively, but the existing test data expects them to be undefined.

Your code added definitions for these values now, so this seems like the test data (json file) just needs to be updated to include the new values.

File: includes/definitions/quantum.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+os: quantum
+type: server
+text: Quantum
+mib_dir: adic
+ifXmcbc: true
+ifname: true
+poller_modules:
```
Comment (murrant) [inline includes/definitions/quantum.yaml:7]: Disabling poller modules is not what you want.  They should be skipped if there is no discovered data for the devices anyway.

**Comment by electrocret** [comment]
> Correct @electrocret (maybe the docs need extra clarification).
> 
 Hint hint, nudge nudge lol. 
I'll look over the docs and update whenever I have time. :)

**Comment by murrant** [comment]
@mpikzink FYI, in upstream several unused port fields have been deleted too.

**Review by murrant — APPROVED** [review]

**Pull request merged**